lib/win_file.dart

```diff
@@ -18,9 +18,26 @@ class WinServer {
     final buffer = byteData.buffer;
     String? tempPath = await _getTemporaryPath();
     if (tempPath == null) throw Exception("Could not get temporary path");
-    var filePath = tempPath + '/file_01.tmp';
-    return File(filePath).writeAsBytes(
-        buffer.asUint8List(byteData.offsetInBytes, byteData.lengthInBytes));
+    var initPath = tempPath + '/file_01.tmp';
+    var filePath = initPath;
+
+    //Prevent multiple applications and file being occupied, max 10
+    for (int i = 1; i < 10; i++) {
+      var file = File(filePath);
+      if (file.existsSync()) {
+        try {
+          file.deleteSync();
+        } catch (e) {
+          filePath = "$initPath$i";
+          continue;
```

Comment:
is it creating multiple files for you ?
like file_011.tmp, file_012.tmp ??

Reply:
yes, like  file_01.tmp1, file_01.tmp2, ... file_01.tmp9,

Reply:
@lunxinfeng i think it would be better to keep it simple, and just delete the file_01.tmp, or we might need to change the name to something like `win_ble_server.exe` maybe instead of file_01.tmp
and only this code would be enough 

if(file.existsSync()) , then file.deleteSync();

Reply:
If I have multiple applications that use win_ble, when I open the second application, the delete will fail because it used by the first application.

Reply:
If the deletion is successful, it means that the file is not occupied and you can use this file name directly. It will only create 1 new file, not 10.

Reply:
i see, ok make sense